pkg/plugins/policies/meshmetric/plugin/v1alpha1/plugin.go

```diff
@@ -290,5 +290,6 @@ func filterPrometheusBackends(backends *[]api.Backend) []*api.PrometheusBackend
 }
 
 func backendNameFrom(endpoint string) string {
-	return strings.ReplaceAll(endpoint, ":", "")
+	// we need to remove "/" as this name will be used as directory name
```

Comment:
Should we just allow [POSIX](https://en.wikipedia.org/wiki/POSIX) "Fully portable filenames" `A–Z a–z 0–9 . _ -` from - https://en.wikipedia.org/wiki/Filename#Comparison_of_filename_limitations

Reply:
this is partially fixed by [updated validation](https://github.com/kumahq/kuma/pull/9634). I am now replacing "/" and ":", so it should now work with valid url without schema